chainweb.openapi.yaml

```diff
@@ -1743,7 +1802,6 @@ paths:
             x-chainweb-node-version:
               $ref: '#/components/headers/x-chainweb-node-version'
         "400":
-          summary: Bad request
```

Comment:
Could that be kept as description. Not all API spec users may remember the meaning of all HTTP status codes. So, adding it may help the reader.